public/static/js/task_admin.js

```diff
@@ -38,6 +38,15 @@ mciModule.controller('AdminOptionsCtrl', ['$scope', '$window', '$rootScope', 'mc
         }
     }
 
+    function doModalFailure(message, data, reload){
```

Comment:
What's the data parameter used for?

Reply:
Nothing- I modeled this on the modal success function, which does use the data parameter, and forgot to remove it when I was done writing my function. I have now removed that parameter.